### PR TITLE
fix: change when express of vspacecode.space

### PR DESCRIPTION
**Pull request by gquittet**
I can't trigger Which Key (pressing space) on Welcome Screen and others screens that are not editors.

To fix this issue I edited the when expression of VSpacecode: Show VSpaceCode Menu

from

sideBarFocus && !inputFocus && !whichkeyActive

to

!inputFocus && !whichkeyActive

Related to https://github.com/VSpaceCode/VSpaceCode/issues/197

### CODE REVIEW

File: src/configuration/keybindings.jsonc [hunk 1]
```diff
@@ -130,4 +130,4 @@
         "command": "file-browser.stepIn",
         "when": "inFileBrowser"
     }
-]
\ No newline at end of file
+]
```
Comment (stevenguh) [inline src/configuration/keybindings.jsonc:133]: nit: remove unnecessary change

File: src/configuration/keybindings.jsonc [hunk 2]
```diff
@@ -9,7 +9,7 @@
     {
         "key": "space",
         "command": "vspacecode.space",
-        "when": "sideBarFocus && !inputFocus && !whichkeyActive"
+        "when": "!inputFocus && !whichkeyActive"
```
Comment (stevenguh) [inline src/configuration/keybindings.jsonc:12]: I am a little bit hesitant to use this because this binding is specifically targeted to sidebar. The changes might have unintended trigger, like user won't able to use space key in custom view.

Another direction to this fix is to find a better `when` clause like line 6 to focus on particular view.

File: src/configuration/keybindings.jsonc (hunk 1 above)
Comment (gquittet) [reply src/configuration/keybindings.jsonc]: Done ✅

**Comment by stevenguh** [comment]
Closing this due to stale PR. Welcome to resubmit if there's a better fix.
